src/base/utils/string.h

```diff
@@ -49,6 +47,14 @@ namespace Utils
         bool naturalCompareCaseInsensitive(const QString &left, const QString &right);
 
         QString wildcardToRegex(const QString &pattern);
+
+        template <typename T>
+        T unquote(const T &str)
+        {
+            if (str.startsWith('\"') && str.endsWith('\"'))
```

Comment:
>Sorry, I can't follow you. Why it is so?

Just to be clear,
I meant `Utils::Net::unquote` is the one in this PR, I propose it moving it under `Utils::Net` for http/network use only.

>because cookies can not be single quoted?

yes, it cannot.